libmamba/src/core/repo.cpp

```diff
@@ -329,7 +330,14 @@ namespace mamba
 
         if (name() != "installed")
         {
-            write_solv(solv_file);
+            try
+            {
+                write_solv(solv_file);
+            }
+            catch (const std::exception& e)
+            {
+                LOG_INFO << "Could not write libsolv solv file for " << json_file << ": " << e.what();
```

Comment:
I think we can emit a warning instead (and to be consistent with mamba 2):
```suggestion
                LOG_WARNING << "Could not write libsolv solv file for " << json_file << ": " << e.what();
```